### ISSUE TITLE
tart 1.0.1 error dialog on start.

### ISSUE THREAD

**Issue by patricks** [cirruslabs/tart#438]
I have updated to version 1.0.1 via homebrew. Now I get the following error dialog if I try to start `tart`.

<img width="372" alt="Screenshot 2023-03-06 at 08 14 07" src="https://user-images.githubusercontent.com/132167/223042858-99a63d6c-0f00-4b1f-9c63-2bb7169b1d13.png">

I have also tried to uninstalling and installing it again.

I am running macOS 13.2.1 (22D68)

If I check the code signing from: `/opt/homebrew/Cellar/tart/1.0.1/libexec/tart.app/Contents/MacOS/tart`

I get the following result:
```
codesign --verify --deep --strict --verbose=2 tart
tart: code has no resources but signature indicates they must be present
```

Is there any way to fix this?

**Comment by edigaryev** [comment]
Hello Patrick! 👋 

It seems that this was introduced in `1.0.1` while `1.0.0` was working just fine. The code signing probably fell off after https://github.com/cirruslabs/tart/pull/436.

You can work around this by navigating to **Settings** → **Privacy & Security**, scrolling down and allowing the `tart` binary to open after you've unsuccessfully ran Tart from the command-line:

<img width="692" alt="Screenshot 2023-03-06 at 14 44 31" src="https://user-images.githubusercontent.com/85709/223090261-7af19c54-6002-4e14-8b5b-848d096d955a.png">

**Comment by patricks** [comment]
I am now on `1.0.3` but I still get the same error.

**Comment by patricks** [comment]
Fixed with `1.0.5` thx! 🥳
